index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="user-login">
+        <input type="text" value="" id="city" placeholder="Enter a city name" />
```

Comment:
You don't need to set the value `value=""`, you could use defaultValue attribute though if you want to have a default value